### ISSUE TITLE
Where are testA and testB ?

### ISSUE THREAD

**Issue by nightbreezecool** [kkakkkka/ETRIS#9]
Does the **test set** used in the experiment combine **testA** and **testB**? Is the **IOU** result output by **test.py** the mean value of testA and testB?

**Comment by kkakkkka** [comment]
No, it's not. You can make corresponding modifications to the scripts in `run_scripts` to calculate the IOU of different subsets. Here is a code implementation that may help you better understand the distribution of subsets in different datasets

```
info = {
    'refcoco': {
        'train': 42404,
        'val': 3811,
        'val-test': 3811,
        'testA': 1975,
        'testB': 1810
    },
    'refcoco+': {
        'train': 42278,
        'val': 3805,
        'val-test': 3805,
        'testA': 1975,
        'testB': 1798
    },
    'refcocog_u': {
        'train': 42226,
        'val': 2573,
        'val-test': 2573,
        'test': 5023
    },
    'refcocog_g': {
        'train': 44822,
        'val': 5000,
        'val-test': 5000
    }
}

```

**Comment by nightbreezecool** [comment]
Is the value of MIoU in the article the mean of the IoU of the data sets val, testA and testB?

**Comment by nightbreezecool** [comment]
Then, is there any part of the project that calculates the loss of the validate set?

**Comment by kkakkkka** [comment]
Hello,
Sorry for the late reply. The article uses `mIoU` as the metric. Currently, our project does not support the calculation of loss for the validation set. However, you can add this by modifying the validate function in the `engine/engine.py` at line 84, and you can refer to the implementation in the `train` function in the `engine/engine.py` at line 42 for reference.
